EU4toV2/Source/V2World/V2World.h

```diff
@@ -117,7 +116,7 @@ class V2World
 		void addUnions();
 		void convertArmies(const EU4::world& sourceWorld);
 
-		void output() const;
+		void output();
```

Comment:
Was there an error that forced you to remove this? If so, that indicates the relevant functionality should be pulled into a separate function and done before outputting.

Reply:
yes, trying to import a value from outside demanded function not be const:
	auto potentialGPs = countCivilizedNations();
I have pulled this outside and passed it along as a variable, returning output() to const.

Reply:
Aha!
countCivilizedNations() could also be made a const function, as it changes nothing.